draft-packages/collapsible/KaizenDraft/Collapsible/Collapsible.module.scss

```diff
@@ -61,25 +62,23 @@ $border-color: rgba($color-gray-600-rgb, 0.2);
 
 .button {
   display: flex;
-  width: 100%;
   align-items: center;
   text-align: left;
   background-color: white;
   border: none;
   transition: background-color $ca-duration-rapid;
-
-  @include ca-padding(
-    $top: $ca-grid,
-    $end: $ca-grid,
-    $bottom: $ca-grid,
-    $start: $ca-grid
-  );
+  padding: $spacing-sm $spacing-md;
```

Comment:
How come these values aren't matching anymore?

Reply:
It's because the IconButton has padding (for the hover/focus styling), whereas the Icon didn't have any.

Reply:
Hmm I've noticed that the chevron has moved a lot further to the left than before now though. Do we need to check with @jmitsios?

Reply:
@HeartSquared - You're right, it looks like it has moved to the left _(unless it's always been that way and gone unnoticed til now?)_ 

Ideally I'd prefer consistent padding around it, if we could reduce the right margin to `12px`?
![image](https://user-images.githubusercontent.com/95595231/189816035-6ca22562-7986-4ce2-8537-2c8f63e1d176.png)